src/pool/srv_pool.c

```diff
@@ -7350,7 +7351,7 @@ pool_svc_update_map(struct pool_svc *svc, crt_opcode_t opc, bool exclude_rank,
 
```

Comment:
probably this(L7346)  should be changed as  this?

if (opc == MAP_EXCLUDE && !(entry->dpe_val & (DAOS_SELF_HEAL_AUTO_REBUILD | DAOS_SELF_HEAL_DELAY_REBUILD)) {
               
}
 So only exclude operation will be checked by this property. Though this is not this patch issue, probably can be done in a different patch.